move-vm-backend/src/types.rs

```diff
@@ -1,3 +1,4 @@
+use alloc::string::String;
```

Comment:
Nice! I think this is my favourite import! It allows programs to speak in a _normal language_ which common non-IT people can understand!